pkg/cmd/ai/add/add.go

```diff
@@ -0,0 +1,41 @@
+package add
+
+import (
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdAIAdd(f *factory.Factory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add",
```

Comment:
I think this should follow the `config` command instead which is used to add the API token for BK

Reply:
It does?

Reply:
Oh I see, I got it mixed up. So this adds `bk ai config add` right?